test/utils.cpp

```diff
@@ -373,6 +373,26 @@ TEST_CASE("extract_filter()", "[utils]")
 
 	REQUIRE(filter == "~/bin/script.sh");
 	REQUIRE(url == "https://newsboat.org");
+
+	utils::extract_filter("https://newsboat.org", filter, url);
```

Comment:
@rnestler Note that this is not possible and will break the rust code. It must contains `filter:`, the C++ code already checks for this. I will remove it for now.